spec/controllers/spree/base_controller_spec.rb

```diff
@@ -9,7 +9,7 @@ def index; authorize!(:read, :something); end
     end
 
     before do
-      stub_spree_preferences(Spree::Config, redirect_back_on_unauthorized: true)
+      allow(Spree::Auth::Engine).to receive(:redirect_back_on_unauthorized?).and_return(true)
```

Comment:
Why is it needed?

Reply:
We need the stub (and the whole spec) to test `config/initializers/solidus_auth_devise_unauthorized_redirect.rb`.